e2e/updatecli.d/success.d/githubrelease.yaml

```diff
@@ -0,0 +1,12 @@
+sources:
+  latestVersion:
+    name: Get latest Venom release
+    kind: githubrelease
+    spec:
+      owner: updatecli
+      repository: updatecli
+      token: '{{ requiredEnv "UPDATECLI_GITHUB_TOKEN" }}'
+      username: '{{ requiredEnv "UPDATECLI_GITHUB_ACTOR" }}'
+      originalVersion: true
```

Comment:
Shouldn't this field name lowercase only?

Also, WDYT of `keeporiginalversion`?

Reply:
I prefer `keeporiginalversion` which I think is more explicit of what it does